sql/1.107/contributor_activity.sql

```diff
@@ -0,0 +1,12 @@
+BEGIN;
+
+CREATE TABLE contributor_activity (
+    id serial NOT NULL PRIMARY KEY,
+    github_id text NOT NULL UNIQUE,
```

Comment:
Could we make `github_id` more descriptive, e.g. `github_event_id` or something?